ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/DefaultErrorMessages.java

```diff
@@ -19,6 +19,15 @@
 
 public class DefaultErrorMessages implements ErrorMessages {
 
+  static String COMMAND_RUNNER_DEGRADED_ERROR_MESSAGE =
```

Comment:
I'd drop the references to `CommandRunner` or command topic here as they are implementation details of the server that we shouldn't leak in the error message. How about something like:
`Failed to process KSQL statement. The server has encountered an incompatible entry in its log and cannot process further statements. This is most likely due to the service being rolled back to an earlier version.`

Reply:
I've updated the error message. I'm going to reuse `COMMAND_RUNNER_DEGRADED_ERROR_MESSAGE` for adding warnings later on to all responses from the `KsqlResource` so I moved "Failed to process KSQL statement" to the `DistributingExecutor`.